### ISSUE TITLE
could not initialize audio. RESULT = 0 when starting server in supercollider

### ISSUE THREAD

**Issue by ionakathryn** [jackaudio/jack2#312]
Hi I'm trying to play some audio with supercollider and when running:
`Server.local.boot;`
getting the error message below:
```
JACK compiled with System V SHM support.
loading driver ..
creating alsa driver ... hw:0|hw:0|1024|2|48000|0|0|nomon|swmeter|-|32bit
could not initialize audio.
RESULT = 0
```
The image shows my jackd settings.
![screenshot from 2017-12-03 16-28-43](https://user-images.githubusercontent.com/16601596/33527374-7b4e6932-d847-11e7-833c-6fd9e0415978.png)

Hopefully someone can help, thanks

**Comment by dvzrv** [comment]
Is this using jack2 or jack1?

I usually start jack2 beforehand and then connect to it with SuperCollider.
However it should work from any jack client requesting the start of the audio server.
So this is not really a SuperCollider specific problem.

Are you able to start JACK with qjackctl, which you have opened in the picture, that you attached?

**Comment by FedericoCapaldo** [comment]
# A possible solution

### TL;DR
Stop your sound system daemon (e.g. `pulseaudio`) so that `jackd` can use the playback device when starting the supercollider server. 
Command to kill pulseaudio on Command Line `pulseaudio -k`. Then try to start supercollider server. 

### Longer version
I had exactely the same erros when trying to start the server on supercollider with `s.boot`.

when trying to start supercollider server, it uses the `JACK Audio Connection Kit sound server` (i.e. `jackd` or `jack1` etc..) to output audio.

I tried to see start jackd manually from the CLI with the command `jackd -r -d alsa` and it gave me the following error.

```jackd 0.125.0
Copyright 2001-2009 Paul Davis, Stephane Letz, Jack O'Quinn, Torben Hohn and others.
jackd comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome to redistribute it
under certain conditions; see the file COPYING for details

could not open driver .so '/usr/lib/jack/jack_net.so': libcelt0.so.2: cannot open shared object file: No such file or directory

could not open driver .so '/usr/lib/jack/jack_firewire.so': libffado.so.2: cannot open shared object file: No such file or directory

JACK compiled with System V SHM support.
loading driver ..
creating alsa driver ... hw:0|hw:0|1024|2|48000|0|0|nomon|swmeter|-|32bit


ATTENTION: The playback device "hw:0" is already in use. Please stop the application using it and run JACK again
cannot load driver module alsa
```

Note at the end that `hw:0` is ALREADY IN USE. 

**My solution and idea** was that the playback device is already being used by your sound system. In my linux version is `pulseaudio`. By killing its daemon with `pulseaudio -k`, then I was able to start the the supercollider server.

**Comment by ZeCraum** [comment]
I had the same problem, but I solved it installing and initializing jack audiolike this:

`jackd1 -r -d alsa -r 44100 `
`jackd2 -r -d alsa -r 44100 `

**Comment by 7890** [comment]
Archiving this hint.

**Comment by josiest** [comment]
I have a slightly different problem than @FedericoCapaldo . `jackd` by default uses `hw:0` playback device. However, when I list my playback devices via `aplay -l`, I get the following output:

```console
card 0: NVidia [HDA NVidia], device 3: HDMI 0 [HDMI 0]
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: NVidia [HDA NVidia], device 7: HDMI 1 [HDMI 1]
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: NVidia [HDA NVidia], device 8: HDMI 2 [HDMI 2]
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: NVidia [HDA NVidia], device 9: HDMI 3 [HDMI 3]
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: NVidia [HDA NVidia], device 10: HDMI 4 [HDMI 4]
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 2: Generic [HD-Audio Generic], device 0: ALC1220 Analog [ALC1220 Analog]
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 2: Generic [HD-Audio Generic], device 1: ALC1220 Digital [ALC1220 Digital]
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 3: USB [Scarlett 2i2 USB], device 0: USB Audio [USB Audio]
  Subdevices: 1/1
  Subdevice #0: subdevice #0
```

That is, `hw:0` is my _graphics card_. `hw:2` is my _built-in audio_ where I have my headphones plugged in. I'm currently trying to figure out how to tell this to `jackd`. If on the rare chance that someone sees this in the mean time and has ideas on how to do this - please share!
